onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -273,9 +273,10 @@ void OpTester::ExecuteModel(Model& model,
                             std::unordered_map<std::string, MLValue> feeds,
                             std::vector<std::string> output_names,
                             const std::string& provider_type) {
-  std::stringstream s1;
-  model.ToProto().SerializeToOstream(&s1);
-  auto status = session_object.Load(s1);
+  std::string s1;
+  model.ToProto().SerializeToString(&s1);
+  std::stringstream sstr(s1);
+  auto status = session_object.Load(sstr);
```

Comment:
Why just load from the protobuf object(model) directly?

Reply:
Because inference session doesn't have a public method to load from modelproto or model object.